pkg/mounter/utils/mounter_utils.go

```diff
@@ -27,22 +27,18 @@ type MounterOptsUtils struct {
 }
 
 func (su *MounterOptsUtils) FuseMount(path string, comm string, args []string) error {
-	klog.Info("-fuseMount-")
-	klog.Infof("fuseMount args:\n\tpath: <%s>\n\tcommand: <%s>\n\targs: <%s>", path, comm, args)
-	cmd := command(comm, args...)
-	err := cmd.Start()
-
-	if err != nil {
-		klog.Errorf("fuseMount: cmd start failed: <%s>\nargs: <%s>\nerror: <%v>", comm, args, err)
-		return fmt.Errorf("fuseMount: cmd start failed: <%s>\nargs: <%s>\nerror: <%v>", comm, args, err)
-	}
-	err = cmd.Wait()
+	klog.Info("-FuseMount-")
+	klog.Infof("FuseMount params:\n\tpath: <%s>\n\tcommand: <%s>\n\targs: <%s>", path, comm, args)
+	out, err := command(comm, args...).CombinedOutput()
 	if err != nil {
-		// Handle error
-		klog.Errorf("fuseMount: cmd wait failed: <%s>\nargs: <%s>\nerror: <%v>", comm, args, err)
-		return fmt.Errorf("fuseMount: cmd wait failed: <%s>\nargs: <%s>\nerror: <%v>", comm, args, err)
+		if mounted, err1 := isMountpoint(path); err1 == nil && mounted { // check if bucket already got mounted
+			klog.Infof("bucket is already mounted using '%s' mounter", comm)
+			return nil
+		}
+		klog.Errorf("FuseMount: command execution failed: <%s>\nargs: <%s>\nerror: <%v>\noutput: <%v>", comm, args, err, string(out))
```

Comment:
Does string(out) always have a value in case of error? else we may hit nil pointer?

Reply:
out is of type `[]byte` so if it does not contain any value, it will convert to empty string